tests/integrations/django/test_basic.py

```diff
@@ -1158,240 +1126,3 @@ def dummy(a, b):
-    assert second_event["spans"][0]["op"] == "cache.get_item"
-    assert second_event["spans"][0]["description"].startswith(
-        "get views.decorators.cache.cache_header."
-    )
-    assert second_event["spans"][0]["data"] == ApproxDict({"cache.hit": False})
-
-    assert second_event["spans"][1]["op"] == "cache.get_item"
-    assert second_event["spans"][1]["description"].startswith(
-        "get views.decorators.cache.cache_page."
-    )
-    assert second_event["spans"][1]["data"]["cache.hit"]
-    assert "cache.item_size" in second_event["spans"][1]["data"]
-
-
-@pytest.mark.forked
-@pytest_mark_django_db_decorator()
-@pytest.mark.skipif(DJANGO_VERSION < (1, 9), reason="Requires Django >= 1.9")
-def test_cache_spans_templatetag(
-    sentry_init, client, capture_events, use_django_caching
-):
-    sentry_init(
-        integrations=[
-            DjangoIntegration(
-                cache_spans=True,
-                middleware_spans=False,
-                signals_spans=False,
-            )
-        ],
-        traces_sample_rate=1.0,
-    )
-    events = capture_events()
-
-    client.get(reverse("view_with_cached_template_fragment"))
-    client.get(reverse("view_with_cached_template_fragment"))
-
-    (first_event, second_event) = events
-    assert len(first_event["spans"]) == 1
-    assert first_event["spans"][0]["op"] == "cache.get_item"
-    assert first_event["spans"][0]["description"].startswith(
-        "get template.cache.some_identifier."
-    )
-    assert first_event["spans"][0]["data"] == ApproxDict({"cache.hit": False})
-
-    assert len(second_event["spans"]) == 1
-    assert second_event["spans"][0]["op"] == "cache.get_item"
-    assert second_event["spans"][0]["description"].startswith(
-        "get template.cache.some_identifier."
-    )
-    assert second_event["spans"][0]["data"]["cache.hit"]
-    assert "cache.item_size" in second_event["spans"][0]["data"]
-
-
-@pytest.mark.parametrize(
-    "method_name, args, kwargs, expected_description",
-    [
-        ("get", None, None, "get "),
-        ("get", [], {}, "get "),
-        ("get", ["bla", "blub", "foo"], {}, "get bla"),
-        (
-            "get_many",
-            [["bla 1", "bla 2", "bla 3"], "blub", "foo"],
-            {},
-            "get_many ['bla 1', 'bla 2', 'bla 3']",
-        ),
-        (
-            "get_many",
-            [["bla 1", "bla 2", "bla 3"], "blub", "foo"],
-            {"key": "bar"},
-            "get_many ['bla 1', 'bla 2', 'bla 3']",
-        ),
-        ("get", [], {"key": "bar"}, "get bar"),
-        (
-            "get",
-            "something",
-            {},
-            "get s",
-        ),  # this should never happen, just making sure that we are not raising an exception in that case.
-    ],
-)
-def test_cache_spans_get_span_description(
-    method_name, args, kwargs, expected_description
-):
-    assert _get_span_description(method_name, args, kwargs) == expected_description
```

Comment:
I am assuming all the deleted tests were moved to `test_cache_module.py`, correct?

Reply:
yes, exactly.